pkg/component/runtime/manager.go

```diff
@@ -658,6 +658,15 @@ func (m *Manager) CheckinV2(server proto.ElasticAgent_CheckinV2Server) error {
 		return status.Error(codes.PermissionDenied, "invalid token")
 	}
 
+	// enable chunking with the communicator if the initial checkin
+	// states that it supports chunking
+	runtime.comm.chunkingAllowed = false
+	for _, support := range initCheckin.Supports {
+		if support == proto.ConnectionSupports_CheckinChunking {
+			runtime.comm.chunkingAllowed = true
```

Comment:
very minor: no need to continue with loop right, although it will be very fast

Reply:
Added `break`